Makefile

```diff
@@ -0,0 +1,16 @@
+develop:
+	python setup.py develop
+
+undevelop:
+	python setup.py develop --uninstall
+
+test:
+	flake8 rds-create-freestoragespace-alarms
```

Comment:
flake8 requires a path. `rds-create-freestoragespace-alarms` is not a valid path